services/provider/proto/provider.proto

```diff
@@ -169,6 +169,7 @@ message ReportStatusRequest{
 }
 
 message ReportStatusResponse{
-    // Contains subscription channel of provider operator for client operator to match
-    string desiredClientOperatorChannel = 1;
+    // Contains hash of desired client state
+    string desiredClientState = 1;
```

Comment:
we are sending a representation calculated by provider and client couldn't validate it unlike the existing response like, send the channel and that will be stored on some resource.

this can just be `providerState`.

Reply:
got LGTM in comments and so resolving the conv.